src/terminal/terminal-handlers.c

```diff
@@ -1070,6 +1070,14 @@ int guac_terminal_csi(guac_terminal* term, unsigned char c) {
                         guac_terminal_scrollbar_set_bounds(term->scrollbar,
                                 -guac_terminal_get_available_scroll(term), 0);
 
+                        /* Redraw normal buffer content */
+                        guac_terminal_redraw_default_layer(term);
+                        
+                        /* Clear selection */
+                        term->text_selected = false;
+                        term->selection_committed = false;
+                        guac_terminal_notify(term);
```

Comment:
Is this call needed? There is an overall call to `guac_terminal_notify()` that occurs after any data is written:

https://github.com/apache/guacamole-server/blob/1c1bcc87c8c47d5005d9400c651afa4980e8cc18/src/terminal/terminal.c#L817